types/context.go

```diff
@@ -278,20 +278,20 @@ func (c Context) Value(key interface{}) interface{} {
 
 // KVStore fetches a KVStore from the MultiStore.
 func (c Context) KVStore(key storetypes.StoreKey) KVStore {
```

Comment:
Can we add an improvement changelog? It is nice to have perf improvement in the changelog.

Reply:
Done!